### ISSUE TITLE
Support: Difference of attributes between 2 MISP instance

### ISSUE THREAD

**Issue by Hybrid2288** [MISP/MISP#9034]
### Support Questions

Hello,
I'm facing an issue with 2 MISP instances of mine.

misp B is pulling IOCs from misp A.
Misp A has 189K attributes & Misp B has 198K Attributes.
I was testing in latest version in lab (2.4.170) but same behaviour.

I don't know how misp B doesn't delete the delta ?
Is MISP supposed to work like that ? 
Our Misp B is working like an exact mirror of Misp A (other network, in DMZ).
We are doing a cron job, executing every 15mins to pull ioc from Misp A.
By executing the command:
scl enable rh-php72 "/var/www/MISP/app/Console/cake Server pull 1 1 full " >> /var/log/misp_sync.log

There is no Event in blocking list.

I saw the issue #8977 
![misp_attr](https://user-images.githubusercontent.com/62749953/233342046-998f015a-0079-45e8-a39e-7a07da774ea1.png)
 but I think I have the opposite.

to summarize: 
Same number of Event but Misp B has more attributes than Misp A :/

Hope this clear ! :)

### MISP version

2.4.150

### Operating System

Redhat

### Operating System version

Centos 7

### PHP version

7.2

### Browser

firefox

### Browser version

102.9.0.esr

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
